app/service/file_svc.py

```diff
@@ -87,6 +87,24 @@ def __init__(self, plugins, exfil_dir):
             return name, xor_file(file_name)
         raise FileNotFoundError
 
+    async def save_file(self, name, content, xored=False):
+        """
+        Save a (payload) file to the stockpile
+        :param name: filename
+        :param content: content to save
+        :param xored: whether or not to xor the contents when saving
+        :return: full path of the saved file
+        """
+        filebase='plugins/stockpile/payloads/'
```

Comment:
We shouldn't have it save to stockpile by default - users could exclude stockpile and this function would fail. Use the top level `data` folder and save to a `payloads` folder there

Reply:
Also, make sure the spacing matches other public functions (i.e. `filebase = 'data/payloads'`)

Reply:
I can do that, but it's a bit harder than it looks. The file service currently only looks in the plugins directory, so doing this means I have to overhaul everything about the search functionality.

Reply:
That makes sense, though it makes things a bit trickier because of remote effects. I think I worked out a solution.